src/js/letters/components/AddressSection.jsx

```diff
@@ -29,8 +29,8 @@ class AddressSection extends React.Component {
           <div className="letters-address">{(destination.fullName || '').toLowerCase()}</div>
           <div className="letters-address">{addressLines.join('').toLowerCase()}</div>
           <div className="letters-address">{(destination.city || '').toLowerCase()}, {destination.state} {(destination.zipCode || '').toLowerCase()}</div>
-          <h5>Why is this address important?</h5>
-          <div>When you download a letter, it will show this address on it. If this address is incorrect you may want to update it, but your letter will still be valid even with the incorrect address. <a href="https://eauth.va.gov/wssweb/wss-common-webparts/mvc/getPCIUUpdateForm" target="_blank">Update the address that appears on your letter(s)</a>.</div>
+          <div>EDIT</div>
+          <div>When you download a letter, it will show this address. If this address is incorrect you may want to update it, but your letter will still be valid even with the incorrect address.</div>
```

Comment:
I think you need a comma after the conditional statement: `If this address is incorrect, you may want to update it`.